gtests/net/mptcp_unsupported/mptcp_ip_tos_server.pkt

```diff
@@ -0,0 +1,15 @@
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
++0      socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0      setsockopt(3, SOL_IP, IP_TOS, [4], 4) = 0
++0.200  getsockopt(7, SOL_IP, IP_TOS, [4], [4]) = 0 
+
++0  bind(3, ..., ...) = 0
++0  listen(3, 1) = 0
+
++0.0  <  [tos 0x04]  S   0:0(0)                    <mss 1460, sackOK, TS val 100        ecr 0,   nop, wscale 8, mpcapable v1 flags[flag_h] nokey>
++0.0  >              S.  0:0(0)  ack 1  win 65535  <mss 1460, sackOK, TS val 700        ecr 100, nop, wscale 8, mpcapable v1 flags[flag_h] key[skey=2]>
```

Comment:
please note, the [tos 0x4] is validated on outbound packet: unless you add it, this test will pass also on unpatched kernels (because the default tos is 0x0)

Reply:
so we should have the opposite I guess, something like this:

```
+0.0  <              S   0:0(0)                    <mss 1460, sackOK, TS val 100        ecr 0,   nop, wscale 8, mpcapable v1 flags[flag_h] nokey>
+0.0  >  [tos 0x04]  S.  0:0(0)  ack 1  win 65535  <mss 1460, sackOK, TS val 700        ecr 100, nop, wscale 8, mpcapable v1 flags[flag_h] key[skey=2]>
+0.0  <               .  1:1(0)  ack 1  win 256    <nop,      nop,    TS val 3081837704 ecr 700,                mpcapable v1 flags[flag_h] key[ckey, skey]>

Reply:
ah got it. We need to set it in an outbound packet. Thanks @dcaratti @matttbe